spec/requests/providers/address_selections_spec.rb

```diff
@@ -62,13 +108,16 @@
     end
 
     it 'creates a new address record associated with the applicant' do
-      expect { post_request }.to change { applicant.reload.addresses.count }.by(1)
+      expect { patch_request }.to change { applicant.reload.addresses.count }.by(1)
     end
 
     it 'redirects to next submission step' do
-      post_request
+      patch_request
 
       expect(response).to redirect_to(providers_legal_aid_application_check_benefits_path(legal_aid_application))
     end
+
+    context 'when an address already exists' do
```

Comment:
❓ Are there tests missing for this context or this was just left here by mistake?

Reply:
Test missing!